test/common/network/raw_buffer_socket_test.cc

```diff
@@ -9,7 +9,7 @@ namespace Network {
 
 TEST(RawBufferSocketFactory, RawBufferSocketFactory) {
   TransportSocketFactoryPtr factory = Envoy::Network::Test::createRawBufferSocketFactory();
-  EXPECT_FALSE(factory->usesProxyProtocolOptions());
+  EXPECT_FALSE(factory->implementsSecureTransport());
```

Comment:
shouldn't this be swapped for testing the hash key method?

Reply:
I can add that, too. I added something to recover coverage since the entire file has just one test.